packages/shared/src/components/squads/Details.tsx

```diff
@@ -49,25 +48,27 @@ const getFormData = async (
 
   const input = document.getElementById(squadImageId) as HTMLInputElement;
   const file = input.files[0];
-  const base64 = await blobToBase64(file);
 
-  return { ...current, file: base64 };
```

Comment:
Why is this removed?

Reply:
The type requirement of the mutation is a `File`, here we convert it to `string`, then before sending it, we then again turn it into a `File.

Reply:
Hmm just make sure it doesn't break implementation (as locally the upload is not e2e testable)
And maybe considering this: https://dailydotdev.slack.com/archives/C01L0QXQJNB/p1724993128390989